.github/workflows/load-tests.yml

```diff
@@ -21,7 +27,7 @@ jobs:
   setup-environment:
     timeout-minutes: 30
     runs-on: self-hosted
-    if: ${{ github.actor != 'dependabot[bot]' }}
+    if: ${{ github.actor != 'dependabot[bot]' && (contains(github.event.pull_request.labels.*.name, 'Run Load Tests') || github.event_name == 'push') }}
```

Comment:
it should be enough to have the label check on the setup job here, since the other jobs depend on this